Library/Homebrew/test/support/helper/test_case.rb

```diff
@@ -12,6 +12,20 @@ class TestCase < ::Minitest::Test
     TEST_SHA1   = "deadbeefdeadbeefdeadbeefdeadbeefdeadbeef".freeze
     TEST_SHA256 = "deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef".freeze
 
+    def before_setup
```

Comment:
You know, I tried about an hour to get this to work, but now that I think about it again, the problem is probably that `FSLeakLogger` is using these also instead of `setup` and `teardown`.